server/src/model.ts

```diff
@@ -2455,3 +2457,55 @@ export async function getFolderContent({
     notMe,
   };
 }
+
+// TODO: Currently only accesses your own content. Do we want to change this?
+export async function getParentFolder(id: number, loggedInUser: number) {
+  return await prisma.content.findUniqueOrThrow({
+    where: { id, isDeleted: false, ownerId: loggedInUser },
+    select: {
+      // TODO: Including name here is a workaround that probably doesn't make sense. We need it for the MoveToFolder menu
+      name: true,
+      parentFolderId: true,
+    },
+  });
+}
+
+export async function getMyFolderContentSparse({
```

Comment:
Add a docstring /** ... */ that explains the purpose of this function, and in particular why we have this as well as `getFolderContent` (which is also missing a docstring like most of these functions are).